test/unit/lucid.relations.spec.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
We should fix the code where we have unused expressions, instead of disabling the rule